match/lib/match/git_helper.rb

```diff
@@ -58,8 +58,8 @@ def self.commit_changes(path, message, git_url, branch = "master")
 
         commands = []
         commands << "git add -A"
-        commands << "git commit -m '#{message}'"
-        commands << "git push origin #{branch}"
+        commands << "git commit -m '#{message.shellescape}'"
```

Comment:
I think we need to get rid of the `'` here, since we're already using `shellescape`, right @mfurtak?